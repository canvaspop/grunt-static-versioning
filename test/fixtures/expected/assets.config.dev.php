<?php
return array(
    'static.assets' => array(
        'global' => array(
            'css' => array(
                '/test.src.css.main.css',
            ),
            'js' => array(
                '/test.src.js.file1.js',
                '/test.src.js.file2.js',
                '/test.components.test.test.js',
                '/test.src.js.file3.js',
                '/test.src.js.file4.js',
            ),
        ),
        'all' => array(
            'css' => array(
            ),
            'js' => array(
                '/test.src.js.file1.js',
                '/test.src.js.file2.js',
                '/test.src.js.file3.js',
                '/test.src.js.file4.js',
            ),
        ),
        'skip' => array(
            'css' => array(
            ),
            'js' => array(
                '/test.src.js.file1.js',
                '/test.src.js.file2.js',
            ),
        ),
    )
);
