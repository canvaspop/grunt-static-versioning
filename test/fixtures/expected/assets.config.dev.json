{
  "staticAssets": {
    "global": {
      "js": [
        "/static/js/test.src.js.file1.js",
        "/static/js/test.src.js.file2.js",
        "/static/js/test.components.test.test.js",
        "/static/js/test.src.js.file3.js",
        "/static/js/test.src.js.file4.js"
      ],
      "css": [
        "/static/css/test.src.css.main.css"
      ]
    },
    "all": {
      "js": [
        "/static/js/test.src.js.file1.js",
        "/static/js/test.src.js.file2.js",
        "/static/js/test.src.js.file3.js",
        "/static/js/test.src.js.file4.js"
      ],
      "css": []
    }
  }
}